test/factories/service_areas.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :service_area do
+    partner_id { 1 }
+    neighbourhood_id { 1 }
```

Comment:
think this only needs to be this to use the factory niceness (associations are implied)

```
partner
neighbourhood
```

Reply:
ah i have yet to get to sorting out the factories. this is not a full PR